scripts/install_chrome_for_ci.py

```diff
@@ -28,7 +28,7 @@
 from scripts import common
 
 
-CHROME_VERSION = '88.0.4324.96-1'
+CHROME_VERSION = '89.0.4389.90-1'
```

Comment:
![image](https://user-images.githubusercontent.com/16653571/115204934-ab496b00-a116-11eb-91d7-8faefc79417e.png)
Can you please remove the version number from these^ files?

Reply:
done!